ui/src/join_game/api.ts

```diff
@@ -56,6 +56,49 @@ export function startGame(game_handle: string) {
   };
 }
 
+export function useEscalator(
+  game_state: GameState,
+  connection_status: ConnectionStatus,
+  heister_selected_keyboard: number // HeisterColor
+) {
+  return async (dispatch) => {
+    if (
+      game_state === null ||
+      connection_status !== ConnectionStatus.Connected
+    ) {
+      console.error("Tried to move heister with no game state / connection");
+      return;
+    }
+    var heisters = game_state.getHeistersList();
+    var heister = heisters.find(
+      (h) => h.getHeisterColor() == heister_selected_keyboard
+    );
+    if (heister === undefined) {
+      console.error("Could not find information for heister");
+      return;
+    }
+    var current_position = heister.getMapPosition();
```

Comment:
This will in reality always be populated, so you can just chuck a `!` on the end to tell tsc that and it'll assume it is non null.